src/util/codecs.js

```diff
@@ -121,3 +122,52 @@ export const isLikelyWebmData = (bytes) => {
     (bytes[2] & 0xFF) === 0xDF &&
     (bytes[3] & 0xFF) === 0xA3;
 };
+
+export const isLikelyMp4Data = (bytes) => {
```

Comment:
Long ago in tba this function was called `isLikelyFmp4Data` but I had to change it to the current iteration because it also matched regular mp4 data. Now I am bringing it back as I think it fits our use case here.